docs/glossary.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Glossary"
+slug: "/contributor/catalyst/glossary"
+---
+
+### Catalyst
+
+[Catalyst](https://github.com/decentraland/catalyst) is the name given to the **Decentraland Service Bundle**, each Catalyst [node](https://decentraland.github.io/catalyst-monitor) exposes a set of services that work as the backbone for the platform and also exposes a public [API](https://decentraland.github.io/catalyst-api-specs/). 
+
+### Bootstrap
+
+The moment when the Catalyst is booted, the node will have a `Bootstrapping` until it is ready to serve content. 
+
+### Catalyst Client
+
+[Library](https://github.com/decentraland/catalyst-client) that can be used to interact with the Catalyst servers. You can both fetch data, or deploy new entities to the server you specify.
+
+### Entity
+
+Data Type, defined with [schemas](https://github.com/decentraland/common-schemas), that cannot be stored in the blockchain because of its size. Some of the current [supported types](https://github.com/decentraland/common-schemas/blob/main/src/platform/entity.ts#L13) are the following:
+
+- [Wearable](https://github.com/decentraland/common-schemas/blob/main/src/platform/item/wearable/wearable.ts)
+- [Emote](https://github.com/decentraland/common-schemas/blob/main/src/platform/item/emote/emote.ts)
+- [Scene](https://github.com/decentraland/common-schemas/blob/main/src/platform/scene/scene.ts)
+- [Profile](https://github.com/decentraland/common-schemas/blob/main/src/platform/profile/profile.ts)
+
+### Deployment
+
+Instances of Entities created on the Content Server and identified with a unique [EntityId](https://github.com/decentraland/common-schemas/blob/main/src/platform/entity.ts#L31). Deployments can be of any [Entity Type](https://github.com/decentraland/common-schemas/blob/main/src/platform/entity.ts#L13).
+
+### Failed Deployment 
+
+Instance of an Entity deployed to a Content Server that failed to pass the [Content Validations](https://github.com/decentraland/adr/blob/main/docs/ADR-51-catalyst-content-validations.md). Validations are implemented on the [content validator](https://github.com/decentraland/content-validator) library. 
+
+### Active Entity 
+
+Current Entity content deployed on a pointer, thus all active entities represent the live content of Decentraland: scenes, profiles, wearables, etc. 
+
+### Deployment History 
+
+Each time a new version of a Deployment is done on a `pointer`, the `Active Entity ` content changes and the previous active content passes to be part of the pointer history. The list of deployments done on a pointer order by timestamp represent the history.
+
+### Pointer 
+
+Pointers are the ids or locations to which a Deployment belongs, they change based on the `Entity Type`, for example: 
+
+- **Wearables**: Uses URN as pointers `urn:decentraland:ethereum:collections-v1:dg_atari_dillon_francis:gerald_patchwork_knit_sweater`
+- **Scenes**: Uses an array of parcels as pointers `[(0,0), (0,1), (0,-1)]`
```

Comment:
This is not entirely true. For scenes, the pointer is `x,y`, for e.g. `-10,20`. The list is not a concept really. It only exists on a deployment where you associate a particular content with a list of pointers.